mxcube3/ui/actions/sampleChanger.js

```diff
@@ -65,20 +98,20 @@ export function scan(address) {
       credentials: 'include'
     }).then(response => {
       if (response.status >= 400) {
-        dispatch(setState('ERROR'));
+        // dispatch(setSCState('ERROR'));
         throw new Error(`Error while scanning sample changer @ ${address}`);
       }
 
       response.json().then(contents => { dispatch(setContents(contents)); });
 
-      dispatch(setState('READY'));
+      // dispatch(setSCState('READY'));
     });
   };
 }
 
 export function loadSample(address) {
   return function (dispatch) {
-    dispatch(setState('MOVING'));
```

Comment:
Right after the loading press button the loading modal appears, so there is no need of additional busy indicator